prow/cluster/jobs/istio/operator/istio.operator.master.yaml

```diff
@@ -60,7 +60,7 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - image: golang:1.12.7
+      - image: gcr.io/istio-testing/istio-builder:v20190624-478286dc
```

Comment:
Do we need to also change this one? It is already working, right?

Reply:
curious to see what image we are using - it is clear that few if anyone are aware of what image we are using across the project.